testdata/descriptions.yml

```diff
@@ -3232,11 +3232,15 @@ parameters:
   preference: # preferences for each point, points with larger preference values are more likely to be chosen as exemplars
     type: values
     values: [1.0, None]
-    default: None
+    default: 1.0
 #  affinity: # which affinity to use
 #    type: values
 #    values: [euclidean, precomputed]
 #    default: euclidean
+  random_state: # seed for starting state None=random
+    type: values
+    values: [0, None]
+    default: 0
```

Comment:
random state should not be part of the sampling and is handled by default with a fixed value